app/lib/frontend/handlers/package.dart

```diff
@@ -5,7 +5,7 @@
 import 'dart:async';
 import 'dart:io';
 
-import 'package:_pub_shared/data/advisories_api.dart';
+import 'package:_pub_shared/data/advisories_api.dart' as advisories_api;
```

Comment:
Naming: I think handlers.dart is not specialized to serve only advisories, it may be better to keep the full library alias.
Conflicts: `Package` is a conflict.